pages/common/pyenv.md

```diff
@@ -0,0 +1,27 @@
+# pyenv  
+
+> Switch between multiple versions of Python easily.
+
+- List all available commands:
+
+`pyenv commands`
+
+- List all Python versions known to pyenv:
+
+`pyenv versions`
+
+- Install a Python version:
+
+`pyenv install {{version}}`
+
+- Uninstall a Python version:
+
+`pyenv uninstall {{version}}`
+
+- Set Python version to be used in global:
```

Comment:
Does "global" mean "in all pyenv commands", "globally in the current machine", or something else?